x-pack/plugins/search_notebooks/public/components/notebooks_view.tsx

```diff
@@ -4,31 +4,40 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+
 import React from 'react';
 import { CoreStart } from '@kbn/core/public';
 import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
 import { KibanaThemeProvider } from '@kbn/react-kibana-context-theme';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 
 import { SearchNotebooks } from './search_notebooks';
-import { NotebookListValue } from '../types';
+import { AppMetricsTracker, NotebookListValue } from '../types';
 
 export interface SearchNotebooksViewProps {
   core: CoreStart;
   queryClient: QueryClient;
+  usageTracker: AppMetricsTracker;
   getNotebookList: () => NotebookListValue;
 }
 
 export const SearchNotebooksView = ({
   core,
   queryClient,
+  usageTracker,
   getNotebookList,
-}: SearchNotebooksViewProps) => (
-  <KibanaThemeProvider theme={core.theme}>
-    <KibanaContextProvider services={{ ...core, notebooks: { getNotebookList } }}>
-      <QueryClientProvider client={queryClient}>
-        <SearchNotebooks />
-      </QueryClientProvider>
-    </KibanaContextProvider>
-  </KibanaThemeProvider>
-);
+}: SearchNotebooksViewProps) => {
+  React.useEffect(() => {
+    usageTracker.load('opened_notebooks_view');
```

Comment:
question - do we want count here or loaded? Would it be interesting to know how many times the user visited notebooks?

Reply:
Thats a good question, I was assuming this would be a single load event, and did the same with view specific notebooks, but a count might be more useful.